.github/pull_request_template.md

```diff
@@ -5,26 +5,28 @@
 ## Description
 <!-- Workshop description that you'll use in advertisements -->
 
+
+# Tasks
+More detailed instructions for specific steps here: <https://cct-datascience.github.io/group-procedures/workshops.html>
+
 ## Pre-workshop checklist:
```

Comment:
I think this checklist would be better as a "workshop schedule" issue template (name is a suggestion), with a link in the README

Reply:
Hmm... so the question is, should this issue template live in organization or in workshop-wednesdays?  There's an issue template for workshops in organization that is currently very redundant of this.  I'm not opposed to moving it here instead and simplifying or removing this PR template.